docs/changelog.md

```diff
@@ -0,0 +1,11 @@
+```{include} ../CHANGELOG.md
+:end-before: Changes for the upcoming release can be found
+```
+
```

Comment:
Try this maybe?

````suggestion

```{important} The version marked as __UNRELEASED__ is not yet released and is under active development.
```

````